packages/dds/matrix/src/matrix.ts

```diff
@@ -511,6 +515,7 @@ export class SharedMatrix<T extends Serializable = Serializable>
                             setOp.value,
                             rowHandle,
                             colHandle,
+                            localSeq,
```

Comment:
...and here's the only "real" change: preserving 'localSeq' during resubmission.